packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -62,6 +62,45 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		};
 	}, [clientDebugger, audience, setAllAudienceMembers, setAudienceHistory]);
 
+	return (
+		<_AudienceView
+			clientId={myClientId}
+			allAudienceMembers={allAudienceMembers}
+			myClientConnection={myClientConnection}
+			onRenderAudienceMember={onRenderAudienceMember}
+			audienceHistory={audienceHistory}
+		/>
+	);
+}
+
+/**
+ * {@link _AudienceView} input props.
+ */
+export interface _AudienceViewProps {
+	clientId: string | undefined;
+	allAudienceMembers: Map<string, IClient>;
```

Comment:
This will need to be an `IClient[]`, so when the chrome extension uses this view it can pass the data in as well.

Reply:
I left it `allAudienceMembers: Map<string, IClient>` as `client-debugger-view/AudienceView` has type of `Map<string, IClient>` for its `allAudienceMembers`. Should I convert the type before passing the `allAudienceMemebers` to `_AudienceView`?

Reply:
Yes, specifically `_AudienceView`, the "internal" one, is where I'm suggesting the change. Because the point of this underscore-named view is to be reusable by the debugger when rendered inline, and when rendered in DevTools. Since DevTools will only have access to the array version of this data, we should update this view to take an array, so we can actually reuse it in both places.

Reply:
Changed to `audienceClientMetaData: AudienceClientMetaData[];`.